message_ix_models/data/bilateralize/node_lists/R12_node_list.yaml

```diff
@@ -0,0 +1,121 @@
+# Region code list
```

Comment:
Are these files any different from files like data/node/R12.yaml?

If they aren't, we should remove the duplicates and use existing code that loads and manipulates the existing files.

If they are, please explain how exactly. I would guess we can find ways to incorporate any information needed by .tools.bilateralize into the existing files.